### DISCUSSION TITLE
Disabled TextBox style

### DISCUSSION THREAD

**Discussion by Mikozans** [microsoft/microsoft-ui-xaml#7518] [Q&A]
How can I manage to copy text and scroll it in a readonly textbox? What I want to achieve is just to have a disabled textbox and be able to copy and scroll it's content.

I tried in many different ways. The closest one was playing around with behaviors. Finally made it modifing the default textbox style but I don't like this approach as it's going to be a future issue if it is updated.

Is there an easiest way to accomplish it?

Thanks in advance!

**Comment by StephenLPeters** [comment]
TextBox has an IsReadOnly property: https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.controls.textbox.isreadonly?view=winrt-19041 does that do what you need it to?

**Comment by asierpn** [comment]
What @Mikozans want to say is how to manage to copy text and scroll in a disabled textbox, or how to get a disabled like style in a readonly textbox. 

The problem with the read-only style is that the user cannot distinguish which fields are editable and which are disabled. If you set "IsEnabled=false" the field can be clearly distinguised as disabled but then you cannot select text or scroll multiline text in the textbox.

[a comment by an automated account is not shown]

**Comment by Mikozans** [comment]
> What @Mikozans want to say is how to manage to copy text and scroll in a disabled textbox, or how to get a disabled like style in a readonly textbox.
> 
> The problem with the read-only style is that the user cannot distinguish which fields are editable and which are disabled. If you set "IsEnabled=false" the field can be clearly distinguised as disabled but then you cannot select text or scroll multiline text in the textbox.

Yes, that's exactly what I tried to say before.
